integration-tests/appsec/endpoints-collection/fastify.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const tracer = require('dd-trace')
+tracer.init({
+  flushInterval: 0
+})
+
+const fastify = require('fastify')
+const app = fastify()
+
+// Basic routes
+app.get('/users', async (_, reply) => reply.send('ok'))
+app.post('/users', async (_, reply) => reply.send('ok'))
+app.put('/users/:id', async (_, reply) => reply.send('ok'))
+app.delete('/users/:id', async (_, reply) => reply.send('ok'))
```

Comment:
adding coverage
```suggestion
app.delete('/example/:file(^\\d+).png', async (_, reply) => reply.send('ok'))
```

Reply:
The case to cover this is not exactly as the one you proposed, but it works to tests the route with regex case.